pkg/util/iptables/iptables.go

```diff
@@ -328,10 +328,11 @@ func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...st
 
 	// Sadly, iptables has inconsistent quoting rules for comments.
 	// Just unquote any arg that is wrapped in quotes.
-	argsCopy := make([]string, len(args))
-	copy(argsCopy, args)
-	for i := range argsCopy {
-		unquote(&argsCopy[i])
+	var argsCopy []string
+	for i := range args {
+		tmpField := args[i]
+		unquote(&tmpField)
+		argsCopy = append(argsCopy, strings.Fields(tmpField)...)
```

Comment:
Sorry, this did not explain to me why you're unpacking every word.  Can you show me a literal iptables output string that is driving this?  This code is known to have worked at some point in the past - what is different?